src/components/Calculator.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import '../styles/Calculator.css';
+
+const CalculatorDisplay = () => (
+  <>
+    <h2>Your Math Magician Calculator</h2>
+    <input type="text" placeholder={0} disabled />
+  </>
+
+);
+
+const CalculatorButtons = () => {
+  const buttons = [
+    { id: 'ac', label: 'AC' },
+    { id: 'plus-minus', label: '+/-' },
+    { id: 'percent', label: '%' },
+    { id: 'plus', label: '+' },
+    { id: 'seven', label: '7' },
+    { id: 'eight', label: '8' },
+    { id: 'nine', label: '9' },
+    { id: 'multiply', label: 'x' },
+    { id: 'four', label: '4' },
+    { id: 'five', label: '5' },
+    { id: 'six', label: '6' },
+    { id: 'minus', label: '-' },
+    { id: 'one', label: '1' },
+    { id: 'two', label: '2' },
+    { id: 'three', label: '3' },
+    { id: 'plus', label: '+' },
+    { id: 'zero', label: '0', class: 'span-2' },
+    { id: 'decimal', label: '.' },
+    { id: 'equals', label: '=' },
+  ];
+
+  return (
+    <div className="all-btns">
+      {buttons.map((button) => (
+        <button key={button.id} type="button" className={`grid-item ${button.class}`}>
+          {button.label}
+        </button>
+      ))}
+    </div>
+  );
+};
```

Comment:
- [ ] Let's try to follow the project requirement as that will prepare us to become better developers that can work to project guidelines. For this project, you are required to style your calculator app to match the original design given in the project requirement. Kindly change your design to the original design 👍🏽

**Your Design**

<img width="589" alt="Screenshot 2023-06-06 at 11 00 54 PM" src="https://github.com/ProgramKingAbel/math-magician-app/assets/50941074/c5652259-9679-427b-95b7-0cc5748e87ea">


**The Original Design**

<img width="318" alt="Screenshot 2023-06-06 at 11 01 14 PM" src="https://github.com/ProgramKingAbel/math-magician-app/assets/50941074/3dd83836-f4d9-4a02-8678-bdc8460ccb53">